Atomic/install.py

```diff
@@ -78,9 +85,16 @@ def __init__(self):
         super(Install, self).__init__()
 
     def install(self):
-        debug = self.args.debug
         if self.args.debug:
             util.write_out(str(self.args))
+        storage_set = False if self.args.storage is None else True
+        storage = _storage if not storage_set else self.args.storage
+        if storage == 'ostree':
+            self.args.system = True
```

Comment:
at the moment we have no other way of using ostree if not using all the system stuff, but we probably avoid this for now (storage == ostree implies --system) until we have a clearer idea.

Also, storage==ostree could imply --user if running as non root. if you are fine with it, let's skip this for now and expect an explicit --system for system containers